### ISSUE TITLE
folding (collapse) & unfolding (uncollapse) of dictionaries

### ISSUE THREAD

**Issue by k1moradi** [goldendict/goldendict#1212]
I usually use a lot of different dictionaries in Goldendict, and I set the collapsing threshold to 10 characters so that all the dictionaries show up collapsed in the beginning.

My problem is that the only region to click to collapse and uncollapse the dictionary is the arrow on the right side of the dictionary. I think it will be more natural to make the whole upper menu clickable for this purpose. 

Also, clicking on the dictionary name unfolds the dictionary but clicking again does not unfold it, which seems like a bug to me.

![image](https://user-images.githubusercontent.com/18602635/68090476-2e530480-fe42-11e9-9cb1-e6e741e91bb7.png)

In summary: The red areas are currently clickable for (fold/unfold). I suggest the whole blue area serve this function.
